liquidity/liquidity.go

```diff
@@ -450,6 +450,13 @@ func (m *Manager) autoloop(ctx context.Context) error {
 		// Create a copy of our range var so that we can reference it.
 		swap := swap
 
+		// Check if the parameter for custom address is defined for loop
+		// outs.
+		if m.params.DestAddr != nil {
+			swap.DestAddr = m.params.DestAddr
+			swap.IsWalletAddr = false
```

Comment:
The reason the `swap` accepts the extra flag is because in many cases an address is generated by the caller-code and we don't know whether that was generated on-the-spot via `NextWallerAddr` or if it was passed down by the user via an explicit `DestAddr` flag.